controllers/imagerepository_controller.go

```diff
@@ -867,6 +867,26 @@ func generateQuayRobotAccountName(imageRepositoryName string, isPullOnly bool) s
 	return robotAccountName
 }
 
+// removeDuplicateUnderscores replaces sequence of underscores with only one.
+// Example: ab__cd___e => ab_cd_e
+func removeDuplicateUnderscores(s string) string {
+	var result strings.Builder
```

Comment:
https://pkg.go.dev/regexp#Regexp.ReplaceAll can we use this instead?

Reply:
Yes, we can. Replaced for loop implementation with regexp.

Reply:
thanks